### ISSUE TITLE
Enable feedback system for RSDs. 

### ISSUE THREAD

**Issue by psanchez-wgu** [wgu-opensource/osmt#309]
### Describe the feature request or enhancement you'd like
As an OSMT user, I want to be able to report or submit feedback on RSD recommendations, revisions, notes, etc.

### Additional context
-- Enable view users to submit feedback on individual RSDs. This feedback is then entered into the feedback system.
-- Admins will then Pull/Identify RSDs that require review based on feedback.

**Comment by ottonomy** [comment]
> This feedback is then entered into the feedback system.

@psanchez-wgu is there an existing system either part of OSMT or elsewhere (an email address associated with a server?) or is this intended to be new data stored by the application, with associated interfaces for viewing it?

**Comment by psanchez-wgu** [comment]
@ottonomy No, there is no existing system. This would be more like enabling a ticketing feature in OSMT. In an initial discussion with the OS team, we talked about an anchor tag.  The idea is to have OSMT communicate with whatever technology is chosen for the feedback (ticketing) system - Qualtrics, MS Forms, etc. I hope this makes sense.

**Comment by ottonomy** [comment]
>  In an initial discussion with the OS team, we talked about an anchor tag

👍  I'd interpret that to mean that something would appear on the RSD detail pages like: 
```
<a href="{{feedbackUrl}}" target="_blank">Open an issue about this RSD</a>
```

Perhaps the `feedbackUrl` could be defined in white label settings. If it's a link to a MS Forms/ Google Forms/Qualtrics, I don't think there's anything at the level of URL query parameters that could be prefilled with data from the RSD, such as its URL. It would be up to the submitter to accurately copy a RSD url into the feedback form to let the feedback processors know which RSD was the subject of the question or comment.

Another approach might be to send an email, to account for some ticketing systems that could automatically open a ticket from an email inbox or other orgs that would process feedback out of an email inbox directly:
```
<a href="mailto:{{feedbackEmail}}?subject=RSD Feedback&body=I have feedback about RSD {{urlEncodedRSDUrl}}" target="_blank">
  Open an issue about this RSD
</a>
```

**Comment by JohnKallies** [comment]
I appreciate the feedback, @ottonomy. Thank you. This is untested, but this SO shares a demo implementation using Google Forms
* https://stackoverflow.com/questions/61849716/auto-fill-or-automate-a-google-form
